manifests/cf-manifest/operations.d/750-s3-broker.yml

```
@@ -4,9 +4,9 @@
   path: /releases/-
   value:
     name: s3-broker
-    version: 0.1.16
-    url: https://s3-eu-west-1.amazonaws.com/gds-paas-build-releases/s3-broker-0.1.16.tgz
-    sha1: 3b53994ebd7faf35186e444664c9020f96e4e7c1
+    version: 0.1.17
+    url: https://s3-eu-west-1.amazonaws.com/gds-paas-build-releases/s3-broker-0.1.17.tgz
+    sha1: e7afa91c671385f886cdf1fb2fc26a484dae29c9P
```

```suggestion
    sha1: e7afa91c671385f886cdf1fb2fc26a484dae29c9
```
I think you've added an unintentional P

---

Great catch! That's what reviews are for, eh?! ❤️